stdlib/contrib/bonitoo-io/hex/hex.flux

```diff
@@ -1,7 +1,77 @@
+// Package hex provides functions that perform hexadecimal conversion
+// of `int`, `uint` or `bytes` values to and from `string` values.
+//
+// introduced: 0.131.0
 package hex
 
-
+// int converts a hexadecimal string to an integer.
+//
+// ## Parameters
+//
+// - v: String to convert.
+//
+// ## Examples
+//
+// ```
+// import "contrib/bonitoo-io/hex"
+//
+// hex.int(v: "4d2")
+//
+// // Returns 1234
+//
+// ```
 builtin int : (v: string) => int
+
+// string converts a Flux basic type to a hexadecimal string.
+//
+// The function is similar to `string()`, but encodes int, uint, and bytes
+// types to hexadecimal lowercase characters.
+//
+// ## Parameters
+//
+// - v: Value to convert.
+//
+// ## Examples
+//
```

Comment:
This function in particular has many examples: https://docs.influxdata.com/flux/v0.x/stdlib/contrib/bonitoo-io/hex/string/#examples